tests/tracking/test_mlflow_artifacts.py

```diff
@@ -67,6 +68,12 @@ def artifacts_server():
         process.kill()
 
 
+@pytest.fixture()
+def mlflow_client(artifacts_server):
+    """Provides an MLflow Tracking API client pointed at the local tracking server."""
+    return mock.Mock(wraps=MlflowClient(artifacts_server.backend_store_uri))
```

Comment:
@abbas123456 Do we need this mock here, or would it be sufficient to construct an `MlflowClient` instance and pass `artifacts_server.backend_store_uri` to the constructor in `test_rest_get_model_version_artifact_api_proxied_artifact_root`? The reason I ask is that the test case doesn't seem to rely on `mlflow_client` being a mock, so it seems that we can remove the mocking.